src/components/Banner/Banner.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../styles/themes.scss";
+@import "../../styles/typography.scss";
+
+.banner {
+  display: grid;
+  padding: 24px;
+  border-radius: 8px;
+  grid-gap: 8px;
+  justify-content: center;
+  align-content: center;
+  @include theme-prop(background-color, dark-background-on-secondary-color);
+  @include theme-prop(color, primary-text-color);
+
+  &.image-position__left,
+  &.image-position__right {
+    grid-template-rows: 1fr auto auto 1fr;
+    grid-auto-columns: auto;
+  }
+
+  &.image-position__left {
+    grid-template-areas:
+    'image  ........    ........'
+    'image  title       title'
+    'image  subtitle    subtitle'
+    'image  ........    ........';
+  }
+
+  &.image-position__right {
+    grid-template-areas:
+    '........ ........  image'
+    'title    title     image'
+    'subtitle subtitle  image'
+    '........ ........  image';
+  }
+
+  &.image-position__bottom,
+  &.image-position__top {
+    grid-auto-rows: auto;
+    grid-template-columns: repeat(1, 1fr);
+  }
+
+  &.image-position__top {
+    grid-template-areas:
+    'image'
+    'title'
+    'subtitle';
+  }
+  &.image-position__bottom {
+    grid-template-areas:
+    'title'
+    'subtitle'
+    'image';
+  }
+
+  &--title {
+    margin: 0;
+    grid-area: title;
+    height: 26px;
+    font-size: 18px;
```

Comment:
i would assume that we would use our tyography - not comment to you but for design
https://style.monday.com/?path=/story/foundations-typography--typography